02/app.js

```diff
@@ -1,2 +1,9 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for (let i = 0; i < x; i++) {
+	if(x < result) {
```

Comment:
Czy faktycznie ten raunek jest tutaj potrzebny? Sam w sobie `for` m warunek, który zatrzymuje pętle.